src/pages/Checkin/Checkin.vue

```diff
@@ -44,17 +48,26 @@
                       <br>
                       {{report.highlight}}
                     </p>
+                    <p @click="showData" v-if="canShowmore" class="readColor btn">Read more..</p>
```

Comment:
use bootstrap class instead of using custom class

Reply:
bootstrap is used in next commit